velox/functions/prestosql/aggregates/tests/SetAggTest.cpp

```diff
@@ -413,5 +413,66 @@ TEST_F(SetAggTest, rowCheckNull) {
       "ROW comparison not supported for values that contain nulls");
 }
 
+TEST_F(SetAggTest, maintainOrdering) {
```

Comment:
I wonder why existing tests are not affected by this change? 

Do you only test global aggregations in the new tests?

Reply:
Got it. Thank you for explaining. Should we tests group-by in addition to global agg?

nit: perhaps, rename to inputOrder to indicate that these tests verify that the order of input is preserved.